spec/atom-reporter.coffee

```diff
@@ -197,13 +197,27 @@ class AtomReporter
     time = "0#{time}" if time.length < 3
     @time.textContent = "#{time[0...-2]}.#{time[-2..]}s"
 
+  specTitle: (spec) ->
+    s = spec.suite
+    parentDescs = while s = s.parentSuite
```

Comment:
It looks like the innermost suite (`spec.suite`) will not have its `description` added to the `parentDescs` array. Is that true / intentional? I'm not too familiar with how the specs and suites are represented in Jasmine, but I would think that `spec.suite` would correspond to the innermost `describe` call.